core/inject.js

```diff
@@ -381,12 +376,13 @@ Blockly.init_ = function(mainWorkspace) {
   }
 
   if (options.moveOptions && options.moveOptions.scrollbars) {
+    // TODO change back to addHorizontal=true before merge.
     mainWorkspace.scrollbar =
         new Blockly.ScrollbarPair(
-            mainWorkspace, true, true, 'blocklyMainWorkspaceScrollbar');
+            mainWorkspace, false, true, 'blocklyMainWorkspaceScrollbar');
```

Comment:
This was set temporarily so the hosted playground has single scrollbars.
This line is also the cause of the mocha test failure.

Reply:
Removed